scout_wheels:
  right_front:
    pose: [ 0.232, -0.20825, -0.1 ]
    joint: right_front_wheel_joint
    <<: &wheel_setting
      wheel_radius: 0.08
      pid: { p: 100.0, i: 1.0, d: 0.0, i_max: 0.0, i_min: 0.0, antiwindup: true, publish_state: true }
  left_front:
    pose: [ 0.232, 0.20825, -0.1 ] # x y beta
    joint: left_front_wheel_joint
    <<: *wheel_setting
  left_back:
    pose: [ -0.232, 0.20825, -0.1 ]
    joint: left_back_wheel_joint
    <<: *wheel_setting
  right_back:
    pose: [ -0.232, -0.20825, -0.1 ]
    joint: right_back_wheel_joint
    <<: *wheel_setting
